spec/models/async_transaction/vet360/base_spec.rb

```diff
@@ -11,6 +11,12 @@
              user_uuid: user.uuid,
              transaction_status: 'RECEIVED')
     end
+    let(:transaction2) do
+      create(:email_transaction,
+             transaction_id: '786efe0e-fd20-4da2-9019-0c00540dba4d',
+             user_uuid: user.uuid,
+             transaction_status: 'RECEIVED')
```

Comment:
Small nit.  If this status is a constant in a class, perhaps use that ref here instead of the `'RECEIVED'` string, for future extensibility.

Reply:
I don't believe we have the upstream tx statuses as constants. Just our local statuses (REQUESTED / COMPLETED).